README.md

```diff
@@ -17,9 +17,9 @@ Some things that WIL includes to whet your appetite:
   that save you the work of manually specifying buffer sizes, calling a function twice
   to get the needed buffer size and then allocate and pass the right-size buffer,
   casting or converting between types, and so on.
-- [`include/wil/registry.h`](include/wil/registry.h): Registry watchers that can
-  call a lambda function or callback you provide whenever a certain tree within
-  the Windows registry changes.
+- [`include/wil/registry.h`](include/wil/registry.h): Type-safe functions to read from, write to,
```

Comment:
Generic comment, not specific to this change (& I don't suggest changing it here): we should add line breaks after each sentence instead of concatenating them all together and breaking at some character width in the file (we can still break long sentences if we don't want to rely on word wrap, but we should still break on punctuation). Ultimately this adds a sentence and a word, but it's harder to see that just looking at the diff.